sql/moz-fx-data-shared-prod/telemetry_derived/error_aggregates_v1/metadata.yaml

```diff
@@ -9,28 +9,12 @@ labels:
   incremental: true
 scheduling:
   dag_name: bqetl_error_aggregates
-  # provide this value so that DAG generation does not have to dry run the
-  # query to get it, and that would be slow because main_v4 is referenced
-  referenced_tables:
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v2"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v3"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v4"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v5"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v6"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v7"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v8"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v9"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "core_v10"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "crash_v4"]
-    - ["moz-fx-data-shared-prod", "telemetry_live", "main_v4"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v2"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v3"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v4"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v5"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v6"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v7"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v8"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v9"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "core_v10"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "crash_v4"]
-    - ["moz-fx-data-shared-prod", "telemetry_stable", "main_v4"]
+  # manually depend on previous day's stable tables, because live tables are
+  # used instead of the current day's stable table partitions
+  depends_on:
+    - dag_name: copy_deduplicate
+      task_id: copy_deduplicate_all
+      execution_delta: 24h
+    - dag_name: copy_deduplicate
+      task_id: copy_deduplicate_main_ping
+      execution_delta: 24h
```

Comment:
there was some logic I did not expect in the automatic dependency resolution that was preventing this from waiting for copy_deduplicate. After looking at the query I made it explicitly depend on yesterday's copy_deduplicate jobs, because that's what is actually referenced.

cc @wlach